src/dtgtk/thumbtable.c

```diff
@@ -2397,7 +2387,7 @@ void dt_thumbtable_update_accels_connection(dt_thumbtable_t *table, const int vi
                             g_cclosure_new(G_CALLBACK(_accel_rate), GINT_TO_POINTER(DT_VIEW_REJECT), NULL));
 
     // History key accels
-    if(!(view & DT_VIEW_LIGHTTABLE))
+//    if(!(view & DT_VIEW_LIGHTTABLE))
```

Comment:
In the new framework, shortcuts do not need to be connected and disconnected each time the view is switched. So all of this will be cleaned up in a later refactor. But I'm trying to keep the number of diffs as low as possible as long as the initial merge isn't completed yet.

In theory the test for !lighttable could still be left active, but it made testing easier to have everything connected early on.

On the other hand, the "don't activate the thumbtable history shortcuts in lighttable" is now hardcoded in _find_views_, so I left this here as a reminder to try to think of a nicer solution, when I do that refactor. I'll add that as a comment.